nmmo/core/env.py

```diff
@@ -34,6 +36,8 @@ def __init__(self,
     self._dead_agents = OrderedSet()
     self.scripted_agents = OrderedSet()
 
+    self.packet_manager = PacketManager.create(self.realm)
```

Comment:
could we call this render_packet_manager or render_state. Something that signals that it's to do with rendering, since "packet" isn't typically understood to deal with rendering. Also, can it be private?

Reply:
PacketManager was renamed to ReplayHelper and moved to the realm

So in `Realm.__init__()`: `self._replay_helper = ReplayHelper.create(self)`